web/core/components/gantt-chart/sidebar/modules/sidebar.tsx

```diff
@@ -12,13 +17,14 @@ import { ModulesSidebarBlock } from "./block";
 type Props = {
   title: string;
   blockUpdateHandler: (block: any, payload: IBlockUpdateData) => void;
-  getBlockById: (id: string, currentViewData?: ChartDataType | undefined) => IGanttBlock;
```

Comment:
We removed a required prop here. Are we not using this component anywhere? @rahulramesha

Reply:
Nope, only used in one place but props for that spread through the parents. The build and linting passed on my local. So it shouldn't be an issue

Reply:
Got it. Thanks for the clarification!